graphql/src/main/java/io/micronaut/configuration/graphql/JacksonGraphQLJsonSerializer.java

```diff
@@ -36,17 +34,6 @@ public class JacksonGraphQLJsonSerializer implements GraphQLJsonSerializer {
 
     private final JsonMapper jsonMapper;
 
-    /**
-     * Default constructor.
-     *
-     * @param objectMapper the {@link ObjectMapper} instance
-     * @deprecated Use {@link #JacksonGraphQLJsonSerializer(JsonMapper)} instead
-     */
-    @Deprecated
```

Comment:
You can remove the `@Inject`annotation from the other constructor. 

We need it only when there are multiple constructors. To tell not to use the`@Deprecated`one.